package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "sds-nodejs",
   "description": "A sample that uses Sequential Data Store in a NodeJS environment.",
-  "version": "1.1.3",
+  "version": "1.1.5",
```

Comment:
1.1.5 -> 1.1.6

Reply:
good catch, i think this was a symptom of the merge conflict resolution. Change it twice but needed to change it four times. I got the package and package lock now, so it should be good.